### PR TITLE
SB-68: Show Reservations on Calendar

### PR DESCRIPTION
**Pull request by EzeLamar**
Changes:
* Use the firebase reservation service to show the reservations on the calendar.

### CODE REVIEW

**Comment by EzeLamar** [comment]
[Show Reservations on Reservations Calendar](https://trello.com/c/ljUUFWRc/68-show-reservations-on-reservations-calendar)

File: app/(admin)/reservations/[id]/page.tsx [hunk 1]
```diff
@@ -2,88 +2,92 @@
...
-			},
-		},
-	];
+		}));
+
+	useEffect((): void => {
+		const fetchData = async () => {
+			try {
+				const courtData = await getCourt(params.id);
+				const reservationsData = await getAllReservations();
```
Comment (EzeLamar) [inline app/(admin)/reservations/[id]/page.tsx:43]: This could be executed in párallel using .then().error()

Comment (EzeLamar) [reply app/(admin)/reservations/[id]/page.tsx:43]: Will be fixed in a future PR 🚀

Comment (lubualo) [reply app/(admin)/reservations/[id]/page.tsx:43]: Alright :)

**Review by lubualo — APPROVED** [review]
Great work

**Pull request merged**